src/components/movie-view/movie-view.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+export class MovieView extends React.Component {
+	constructor() {
+		super();
+
+		this.state = {};
+	}
+
+	render() {
+		const { movie, onClick, button } = this.props;
+
+		if (!movie) return null;
+
+		return (
+			<div className="movie-view">
+				<img className="movie-poster" src={movie.ImagePath} />
+				<div className="movie-title">
+					<span className="label">Title: </span>
+					<span className="value">{movie.Title}</span>
+				</div>
+				<div className="movie-description">
+					<span className="label">Description: </span>
+					<span className="value">{movie.Description}</span>
+				</div>
+				<div className="movie-genre">
+					<span className="label">Genre: </span>
+					<span className="value">{movie.Genre.Name}</span>
+				</div>
+				<div className="movie-director">
+					<span className="label">Director: </span>
+					<span className="value">{movie.Director.Name}</span>
+				</div>
+				<div className="back-button">
+					<button onClick={() => window.open('mainView', '_self')} className="button">
```

Comment:
you can try with
`<Link to="" onClick={() => history.back()}>
              <Button type="button" variant="link" size="sm">
                Back
              </Button>
            </Link>`

Reply:
This won't work for now since the app is a single page and doesn't have a different page to go back to.